src/components/Explorer/Base/Result/RunLookup/RunLookup.tsx

```diff
@@ -32,10 +44,21 @@ export const RunLookup = ({ runId, filters }: Props) => {
     React.useEffect(() => {
         if (!runId) return
         getRunTitle(runId).then(setPageTitle)
+        callApi(runId).then((data) => {
+            setTableData(data)
+        })
+        return () => {
+            setTableData([])
+        }
```

Comment:
What's the purpose of this return? If it's just to set initial state, that should go in the `useState`:

```ts
const [tableData, setTableData] = useState<JSON[]>([])
```

Reply:
Oh I see, this is the cleanup function. Made a slight adjustment in 4d8ee25 so it's more clear.